tickets/src/components/content/purchase/PayButton.js

```diff
@@ -24,9 +26,13 @@ export const PayButton = ({ transaction, purchaseKey, config }) => {
         {requiredConfirmations})
       </PayInfo>
     )
-  } else if (transaction && transaction.status === 'mined') {
+  } else if (keyStatus === KeyStatus.CONFIRMING) {
+    // Key is being confirmed but we don't have transaction
```

Comment:
Hum. How is that possible? The transaction should come before the key?

Reply:
I think you should copy the logic from `paywall/src/components/lock/Lock.js` here